app/setup.py

```diff
@@ -342,6 +345,19 @@ def setup_submitter(application):
         raise Exception("Unknown EQ_SUBMISSION_BACKEND")
 
 
+def setup_task_client(application):
+    if application.config["EQ_SUBMISSION_CONFIRMATION_BACKEND"] == "cloud-tasks":
+        queue = application.config["EQ_SUBMISSION_CONFIRMATION_QUEUE"]
+
+        if not queue:
+            raise Exception("Setting EQ_SUBMISSION_CONFIRMATION_QUEUE Missing")
+
+        application.eq["task-client"] = CloudTaskPublisher(queue)
```

Comment:
The queue should not be passed here. The Task publisher can be generic like Pub/Sub then the queue is passed at the time of task creation. Basically the same principle as Pub/Sub.